test/unit/specs/mount/Wrapper/hasClass.spec.js

```diff
@@ -46,4 +46,10 @@ describe('hasClass', () => {
 
     expect(wrapper.hasClass('color-red')).to.equal(true)
   })
+
+  it('returns true when the element contains multiple classes', () => {
```

Comment:
Can you add a test for when wrapper.element is undefined. We want to either handle the error, or returns false. Previous behavior was to return false